assets/src/dashboard/components/button/index.js

```diff
@@ -29,11 +29,11 @@ const StyledButton = styled.button`
   align-items: center;
   color: ${({ theme }) => theme.colors.white};
   cursor: pointer;
-  border: 1px solid transparent;
+  border: 0.1rem solid transparent;
```

Comment:
cc @swissspidy would this be recommended to use rem's under WP? We usually just work with `px` otherwise. Personally I usually struggle with em-based units since they are derived from height, while used for height and width.

Reply:
I don't think WordPress admin uses rem anywhere, and mixing rem with what WordPress admin does might cause lots of issues down the road.